index.html

```diff
@@ -10,28 +10,109 @@
+      <article class="project">
+         <img src="resources/P1-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>               </div>
+            <ul class="buttons-flex">
+            <li class="li-button">html</li>
+            <li class="li-button">css</li>
+            <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt; </button>
+         </div>            
+      </article>
+
+      <article class="project">
+         <img src="resources/P2-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>   
+            </div>
+            <ul class="buttons-flex">
+               <li class="li-button">html</li>
+               <li class="li-button">css</li>
+               <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt;</button>
+         </div>            
+      </article>
+ 
+      <article class="project">
+         <img src="resources/P3-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>   
+            </div>
+            <ul class="buttons-flex">
+               <li class="li-button">html</li>
+               <li class="li-button">css</li>
+               <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt;</button>
+         </div>            
+      </article>
+      <article class="project">
+         <img src="resources/P4-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>   
```

Comment:
- [ ] At the moment we don't need `Lorem Ipsum` we will use it later for the pop-up window. Kindly use the title `Project name goes here` instead. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/185765985-10151159-4701-4e02-a7d6-98b3fa0f3cf3.png)